Paco-Server/src/com/google/sampling/experiential/server/ACLHelper.java

```diff
@@ -0,0 +1,92 @@
+package com.google.sampling.experiential.server;
+
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import com.pacoapp.paco.shared.util.SearchUtil;
+
+import net.sf.jsqlparser.statement.Statement;
+
+public class ACLHelper {
+  public static String getModifiedQueryBasedOnACL(String selectSql, String loggedInUser,
+                                                  List<Long> adminExperimentsinDB) throws Exception {
+    Statement selStat = SearchUtil.getJsqlStatement(selectSql);
+    return getModifiedQueryBasedOnACL( selStat, loggedInUser,adminExperimentsinDB);
+  }
+  public static String getModifiedQueryBasedOnACL(Statement selectSql, String loggedInUser,
+                                                  List<Long> adminExperimentsinDB) throws Exception {
+    String adminExpIdCSV = adminExperimentsinDB.toString();
+    String loggedInUserWithQuotes = "'"+ loggedInUser +"'";
+    boolean ownData = false;
+    boolean ownExpt = false;
```

Comment:
It seems like this variable is closer to the idea of allExperimentsAreAdministeredByRequestor. That is way too long of course. Maybe adminOnAllExperiments?